dataverse-persistence/src/main/resources/Bundle_pl.properties

```diff
@@ -63,7 +63,7 @@ first=Pierwszy
 last=Ostatni
 more=Wi\u0119cej..
 less=Mniej..
-select=Wybierz
+select=Wybierz..
```

Comment:
I think that two dots (here & in the existing cases) are something strange from typographical POV and we should use three (... or – better – as one character: … (RAlt + k on linux))

Reply:
I've used two to be compatible with other cases. I'd rather stick to two now and point this out at the meeting to switch them all in one go if we decide to change it.